resources/chef_org.rb

```diff
@@ -58,11 +58,17 @@
     end
   end
 
-  # TODO: fix idempotency for org admins
+  org_admins = JSON.parse(
```

Comment:
This really shouldn't be done by parsing knife. That's going to be super problematic and brittle. You'll want to hit the server API directly and parse out that output from there.

Reply:
Fair enough. I was sort of following the pattern of parsing output in some other parts of the same resource but I'll rework it to hit the API.